libs/small-steps-test/src/Control/State/Transition/Trace.hs

```diff
@@ -495,7 +495,7 @@ applySTSTest = applySTSOptsEither defaultOpts
   where
     defaultOpts =
       ApplySTSOpts
-        { asoAssertions = AssertionsAll,
+        { asoAssertions = AssertionsOff,
           asoValidation = ValidateAll,
           asoEvents = EPDiscard
```

Comment:
Not sure why this was changed. Not sure I did this.

Reply:
I think you started with my original branch, where I was doing that for debugging purposes. We shouldn't check this in.

Reply:
I set it back to AssertionsAll